tests/unit/parsers/test.manifestjson.js

```diff
@@ -3271,10 +3305,9 @@ describe('ManifestJSONParser', () => {
       expect(linter.collector.errors).toEqual(
         expect.arrayContaining([
           expect.objectContaining({
-            instancePath: '/background',
-            code: 'JSON_INVALID',
+            code: 'MANIFEST_FIELD_UNSUPPORTED',
             message: expect.stringMatching(
-              /"\/background" .* has invalid extra properties/
+              /"\/background\/service_worker" is not supported/
```

Comment:
This error is not [the one we raise from here in manifestjson.js](https://github.com/mozilla/addons-linter/blob/3d5d6ce72994745c3cf48921579873b1f076d52c/src/parsers/manifestjson.js#L576-L585), instead of the generic one that anyOf ajv validation used to raise.